tests/test_bootstrap.py

```diff
@@ -147,7 +152,7 @@ def start_walking(self):
                         self._pcandidates = [self.get_candidate(address) for address in
                                              set(self._dispersy._discovery_community.bootstrap.candidate_addresses)]
                         break
-                    yield deferLater(reactor, 1, lambda: None)
+                    sleep(1)
```

Comment:
What's the reason why the `deferLater` (the one used before your changes) does not work now anymore? This method is still decorated with `@lineCallbacks` right?